### ISSUE TITLE
Training set test set accuracy rate difference is too large

### ISSUE THREAD

**Issue by Kobeee0614** [open-mmlab/mmdetection#794]
Hello, my training set accuracy rate is 90+, test set 60+, is there any relationship with my parameter settings?
Training set 3000 pictures, test set 800 pictures, picture size is 512 * 512, batchsize is 16, learning rate 0.02.

**Comment by hellock** [comment]
Since your training set is small, so this can be possible. The hyper-parameters are not the best for any dataset, so the performance may be improved if you tune the parameters.

**Comment by solauky** [comment]
Meet same problem, how to solve finally excuse me?thank you
